src/constants/default-settings.js

```diff
@@ -789,6 +789,9 @@ export const DEFAULT_NOTIFICATION_TOPICS = keyMirror({
   file: null
 });
 
+// Minimum time between identical notifications about deck.gl errors
+export const THROTTLE_NOTIFICATION_TIME = 200;
```

Comment:
Seems short? I can't imagine anyone wants 5 messages per second

Reply:
I'll increase this time to 1000. This is more to prevent any React related problems, not for a user. 
The error notifications are going to stay or pop up again,  as errors from deck.gl are triggered on every update.